opentok/opentok-android-sdk-samples
# [VIDCS-923] Android MP Video Transformers sample

**What is this PR doing?**

Adds Video transformers sample.

**How should this be manually tested?**

There is a button which toggles the video transformers (background blur and logo).
Once set, it should be visible instantly on the video stream.

https://jira.vonage.com/browse/VIDCS-923
